packages/yoastseo/src/values/Mark.js

```diff
@@ -163,7 +237,18 @@ Mark.prototype.isValid = function() {
  * @returns {boolean} Returns true if the Mark object has position information, false otherwise.
  */
 Mark.prototype.hasPosition = function() {
-	return !! this.getPositionStart && this.getPositionStart();
+	return ! isUndefined( this.getPositionStart() );
```

Comment:
A Mark has position if the position start doesn't return `undefined`. 
Lodash's `isUndefined()` doesn't evaluate `0` as `undefined`. For this reason, this method is used here. This is because `getPositionStart()` can still return `0` which is still a valid position start. However, using the double negation `!!` will evaluate `0` as `false` (see [this article)](https://www.sitepoint.com/javascript-double-negation-trick-trouble/)